src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -126,6 +126,7 @@ private void updateRobotState() {
         getPose()
             .getTranslation()
             .getDistance(FieldConstants.Speaker.centerSpeakerOpening.toTranslation2d());
+    Logger.recordOutput("Drive/SpeakerDistance", distance - 30.2);
```

Comment:
Remove the constant 30.2 fudge factor - we want to either fix this in Photon Vision or just rely on the wrong (bug hopefully repeatable) location value.